src/Player.cpp

```diff
@@ -58,4 +58,25 @@ void Player::setGlobalBottom(float bottom)
     setPosition(bounds.left, y);
 }
 
-} // namespace tt
\ No newline at end of file
+
+void Player::addItem(std::string itemID) {
```

Comment:
Also, I just noticed you're passing the `std::string` by value:

```
void Player::addItem(std::string itemID) 
```

which you probably don't want since it creates an unnecessary copy of `std::string`. Instead use a const-reference:

```
void  Player::addItem(const std::string& itemID)
```

This avoids the extra copy.